pages/api/ws.js

```diff
@@ -259,7 +259,26 @@ const ioHandler = (req, res) => {
             );
           } else if (message.action === "game_window") {
             let [room_code, user_id] = message.session.split(":");
+
+            // First, check if the room exists
+            if(!rooms[room_code]) {
+              ws.send(JSON.stringify({ 
+                action: "error", 
+                code: ERROR_CODES.ROOM_NOT_FOUND 
+              }));
+              return;
+            }
+
```

Comment:
Merge `master` pages/api/ws.js should no longer exist know

Reply:
Tell me if this feature is still relevant with the new backend.

Reply:
Probably better to close this as this is irrelevant now, if statement is out of date as well since host is no longer in registeredPlayers.